### PR TITLE
Dev/linux bringup poc

### PR DESCRIPTION
**Pull request by tomereli**
Bring up controller on Linux native, tested on Ubuntu18.04 with mac80211_hwsim - see README.me for running instructions.

Summary:
- Split BEEROCKS_PATH to BEEROCKS_BIN_PATH and BEEROCKS_CONF_PATH for
Linux build
- fix implicit fallthrough warning in network_map
- add initial README.md with Linux instructions

### CODE REVIEW

File: src/beerocks/master/db/network_map.cpp [hunk 1]
```diff
@@ -755,11 +755,11 @@ std::ptrdiff_t network_map::fill_bml_node_statistics(db &database, std::shared_p
     case beerocks::TYPE_IRE_BACKHAUL: {
         if (database.is_node_wireless(n->mac)) {
             n_type = beerocks::TYPE_CLIENT;
-            // fall to case beerocks::TYPE_CLIENT
         } else {
             break;
         }
     }
+        [[fallthrough]];
```
Comment (arnout) [inline src/beerocks/master/db/network_map.cpp:762]: Has this been clang-formatted? Indentation looks weird.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+The prplMesh Agent implements a Multi-AP v1.0 certifiable controller (work in progress).
+
+# Build
+Build system is CMAKE, and supports RDKB and UGW cross compilation builds.
+Building natively (Linux only) is partially supported only for development process. Some of the components are skipped / built as stubs in this mode.
+
+# Run controller on Linux
+The controller needs to run either on the same host as the agent or in another host with L2 connectivity (provided the BTL is using the local bus and not UDS mode).
+
+Currently, this mode was tested on an Ubuntu18.04 with hwsim, the agent was compiled to UDS mode.
```
Comment (arnout) [inline README.md:14]: agent -> controller

**Review by arnout — DISMISSED** [review]

**Comment by arnout** [comment]
Since it's just one tiny change, you can push yourself after fixing the agent -> controller.

File: README.md (hunk 2 above)
Comment (tomereli) [reply README.md:14]: Done

File: src/beerocks/master/db/network_map.cpp (hunk 1 above)
Comment (tomereli) [reply src/beerocks/master/db/network_map.cpp:762]: It has - indentation looks weird due to the parenthesis which is aligned to the "case", but the code is indented - and seems that `[[fallthrough]]` is considered as code.

**Pull request merged**